tests/transform.spec.mts

```diff
@@ -1,30 +1,31 @@
 import { expect, test } from 'vitest';
-import { flattenDefaultTokens, pxToBaseSize } from '../scripts/transform';
+import { flattenDefaultTokens, pxToBaseSize } from '../scripts/transform.mts';
 import { TransformedToken } from 'style-dictionary';
+import { DesignToken } from 'style-dictionary/types';
 
 const sampleTokens: TransformedToken[] = [
 	{
 		name: 'Background',
 		path: ['color', 'bg', '_'],
 		value: '#ffffff',
 		isSource: false,
-		original: null,
+		original: undefined as unknown as DesignToken,
```

Comment:
Could you please explain this change?

Reply:
Since migrating to Style Dictionary v4, `TransformedToken` requires a `DesignToken` object for the `original` property. It's not needed in the tests, so I'm casting to unknown to make the TS compiler happy.